### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Modullah**
I was unsure whether I was to use 'choco install make' using built in windows shell tools or to use the windows subsytem for linux (bash) to run the install. 
Modified the read me to be more clear.

### CODE REVIEW

**Comment by Modullah** [comment]
Signed CLA and re opening pr.

File: README.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ Install the following tools, if you don't have them already:
   and setup might not work if you use another shell.
 
 - **GNU Make**. 
-  On Windows the easiest way to install Make is `choco install make`. 
-  Other options include using a [subsystem][wsl].
+  On Windows the easiest way to install Make is `choco install make` using 
+  command prompt or powershell as admin. Other options include using a [subsystem][wsl].
```
Comment (parlough) [inline README.md:52]:
```suggestion
  On Windows the easiest way to install Make is `choco install make`
  using command prompt or powershell as as admin. 
  Other options include using a [subsystem][wsl].
```

**Review by parlough — APPROVED** [review]
Glad to have any addition that makes instructions more clear.

One small adjustment to better split the lines, but looks good to me! Thanks :)

File: README.md (hunk 1 above)
Comment (Modullah) [reply README.md:52]: No problem. Was not sure about the spacing of the sentences. Thank you for assisting!

**Comment by parlough** [comment]
Looks good to me, thanks for adjusting this!

**Pull request merged**